WordPress/src/main/java/org/wordpress/android/ui/accounts/BlogUtils.java

```diff
@@ -150,6 +152,10 @@ public static boolean addOrUpdateBlog(String blogName, String xmlRpcUrl, String
                     blog.setPlanShortName(planShortName);
                     blogUpdated = true;
                 }
+                if (!blog.getCapabilities().equals(capabilities)) {
```

Comment:
The rest of the code allows for `null` and so the `blog.getCapabilities()` can return null. Better to null check. Cheers!